python/dgl/frame.py

```diff
@@ -38,6 +38,19 @@ def flatten(self):
             flat_index = F.gather_row(flat_index, index)
         return flat_index
 
+class LazyFeature(object):
+    __slots__ = ['name', 'id_']
+    def __init__(self, name=None, id_=None):
+        self.name = name
+        self.id_ = id_
+
+    def to(self, *args, **kwargs):
+        return self
```

Comment:
`LazyFeature` does not actually contain any feature so it should not involve any copying when a graph is copied to another device.  The actual features will be sliced by calling `FeatureStorage.fetch(LazyFeature.id_, device)` in the DataLoader so there is no inference of the output device.  I put this method here so that `Frame.to()` would not break.